src/loaders/core_text.rs

```diff
@@ -244,6 +244,10 @@ impl Font {
         self.core_text_font.symbolic_traits().is_monospace()
     }
 
+    pub fn is_emoji(&self) -> bool {
+        self.family_name() == "AppleColorEmoji" || self.family_name() == "Apple Color Emoji"
```

Comment:
Is Apple Color Emoji the only font that includes emoji? I have seen other fonts like Noto Color Emoji that could include emojis. There is a list here: https://typography.guru/list/topic/emoji-fonts/

Reply:
this matches what iterm does:

https://github.com/gnachman/iTerm2/blob/c52136b7c0bae545436be8d1441449f19e21faa1/sources/Metal/Support/iTermCharacterSource.m#L639



Added a comment to that effect

Reply:
Thinking about it twice. This makes sense as "Apple Color Emoji" is the only font in fallback list that supports emojis. Since users are not likely to set their primary terminal font to be a emoji font, this should not be an issue.

Reply:
the other way I can think about doing this is the way alacritty does it, which is to check if the glyph is colored by checking this bit: https://developer.apple.com/documentation/coretext/ctfontsymbolictraits/kctfontcolorglyphstrait

(from alacritty:):
```rust
 pub fn is_colored(&self) -> bool {
        (self.ct_font.symbolic_traits() & kCTFontColorGlyphsTrait) != 0
    }
```

Reply:
unresolving just so this is seen

Reply:
done